jira-create-issue/create_issue.js

```diff
@@ -0,0 +1,49 @@
+const core = require('@actions/core');
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
+
+try {
+  const baseUrl = core.getInput('base_url', {required: true});
+  const userEmail = core.getInput('user_email', {required: true});
+  const apiToken = core.getInput('api_token', {required: true});
+  const description = core.getInput('description', {required: true});
+  const issueType = core.getInput('issue_type', {required: true});
+  const priority = core.getInput('priority', {required: true});
+  const project = core.getInput('project', {required: true});
+  const reporter = core.getInput('reporter', {required: true});
+  const summary = core.getInput('summary', {required: true});
+  const labels = core.getMultilineInput('labels', {required: false});
+  const timeout = core.getInput('timeout', {required: false});
+
+  const bodyData = {
+    fields: {
+        description: description,
+        issuetype: { id: issueType },
+        priority: { id: priority },
+        project: { id: project },
+        reporter: { id: reporter },
+        summary: summary,
+        labels: labels
+    }
+  };
+
+  fetch(`${baseUrl}/rest/api/2/issue`, {
+    method: 'POST',
+    headers: {
+      'Authorization': `Basic ${Buffer.from(`${userEmail}:${apiToken}`).toString('base64')}`,
+      'Accept': 'application/json',
+      'Content-Type': 'application/json'
+    },
+    signal: AbortSignal.timeout(Number(timeout)),
+    body: JSON.stringify(bodyData)
+  })
+  .then(response => {
```

Comment:
nitpick: It is pretty much common practice to place the call to the `then` method of a `Promise` on the same line as the closing body of the Promise's function call:
```suggestion
  }).then(response => {
```